content/tracing/proxies/envoy.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Envoy
+kind: Documentation
+further_reading:
+- link: "tracing/visualization/"
+  tag: "Use the APM UI"
+  text: "Explore your services, resources and traces"
+- link: "https://www.envoyproxy.io/"
+  tag: "Documentation"
+  text: "Envoy website"
+- link: "https://www.envoyproxy.io/docs/envoy/latest/"
+  tag: "Documentation"
+  text: "Envoy documentation"
+- link: "https://github.com/DataDog/dd-opentracing-cpp"
+  tag: "Source Code"
+  text: Datadog OpenTracing C++ Client
+---
+
+Support for Datadog APM has been included in Envoy.
+It is available in the `envoyproxy/envoy:latest` docker container, and is included in the upcoming 1.9.0 release.
+
+## Enabling Datadog APM
+
+Three settings are required to enable Datadog APM in Envoy.
```

Comment:
Since we're calling out there being three settings, let's put them in an unordered list